tests/e2e/pkg/common.go

```
@@ -500,6 +500,17 @@ func CheckNSFinalizerRemoved(ns string, clientset *kubernetes.Clientset) bool {
 	return namespace.Finalizers == nil
 }
 
+func getKubeConfigPath(kubeconfigPath string) string {
+	var err error
+	if kubeconfigPath == "" {
+		kubeconfigPath, err = k8sutil.GetKubeConfigLocation()
+		if err != nil {
+			Log(Error, fmt.Sprintf("Failed to get default kubeconfig path: %v", err))
```

Seems like an issue that this failure is logged but not returned.

---

i dont really understand the point of this function, seems the entire point is to swallow the error of another function. I will remove it.